shapely/tests/test_constructive.py

```diff
@@ -972,6 +973,24 @@ def test_minimum_bounding_circle(geometry, expected):
     assert_geometries_equal(actual, expected)
 
 
+@pytest.mark.parametrize(
+    "geometry, expected_type",
+    [
+        (Point(0, 0), LineString),
+        (LineString([(0, 0), (1, 1)]), LineString),
+        (Polygon([(0, 0), (1, 0), (1, 1), (0, 1), (0, 0)]), LineString),
+        (empty_point, LineString),
+        (empty_line_string, LineString),
+        (empty_polygon, LineString),
+    ],
+)
+def test_minimum_width(geometry, expected_type):
+    actual = shapely.minimum_width(geometry)
+    assert isinstance(actual, expected_type)
```

Comment:
```suggestion
    assert isinstance(actual, LineString)
```

Since it is always the same, can hardcode this instead of having it part of the parametrization